bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/HomeConnectHoodHandler.java

```diff
@@ -84,10 +84,16 @@ protected void configureChannelUpdateHandlers(Map<String, ChannelUpdateHandler>
                         if (data.getValue() != null) {
                             boolean enabled = data.getValueAsBoolean();
                             if (enabled) {
-                                Data brightnessData = apiClient.get().getFunctionalLightBrightnessState(getThingHaId());
-                                getThingChannel(CHANNEL_FUNCTIONAL_LIGHT_BRIGHTNESS_STATE)
-                                        .ifPresent(channel -> updateState(channel.getUID(),
-                                                new PercentType(brightnessData.getValueAsInt())));
+                                getThingChannel(CHANNEL_FUNCTIONAL_LIGHT_BRIGHTNESS_STATE).ifPresent(channel -> {
+                                    try {
+                                        Data brightnessData = apiClient.get()
+                                                .getFunctionalLightBrightnessState(getThingHaId());
+                                        updateState(channel.getUID(), new PercentType(brightnessData.getValueAsInt()));
+                                    } catch (CommunicationException | ApplianceOfflineException
+                                            | AuthorizationException e) {
+                                        updateState(channel.getUID(), UnDefType.UNDEF);
```

Comment:
Shouldn't you set the Thing status to OFFLINE here instead?

Reply:
Undone this change